airbyte-integrations/connectors/destination-databricks/BOOTSTRAP.md

```diff
@@ -1,6 +1,6 @@
-# Databricks Delta Lake Destination Connector Bootstrap
+# Databricks Lakehouse Destination Connector Bootstrap
 
-The Databricks Delta Lake Connector enables a developer to sync data into a Databricks cluster. It does so in two steps:
+The Databricks Lakehouse Connector enables a developer to sync data into a Databricks cluster. It does so in two steps:
```

Comment:
Maria wants the doc to mention "Delta Lake on Databricks Lakehouse" in the first sentence.

```suggestion
This destination syncs data to Delta Lake on Databricks Lakehouse. It does so in two steps:
```